src/PhotoNim.nim

```diff
@@ -1,24 +1,28 @@
-import PhotoNim/[geometry, hdrimage, camera]
+import PhotoNim/[geometry, hdrimage, camera, shapes]
 
 import docopt
 from nimPNG import savePNG24
 
 from std/os import splitFile
 from std/streams import Stream, newFileStream, close
-from std/strutils import parseFloat
+from std/strutils import parseFloat, parseInt
 from std/strformat import fmt
-from std/math import pow
-
+from std/math import pow, exp
+from std/times import cpuTime
+import typetraits
 
 let PhotoNimDoc = """PhotoNim: a CPU raytracer written in Nim.
 
 Usage:
     ./PhotoNim pfm2png <input> [<output>] [--alpha=<alpha> --gamma=<gamma>]
-    ./PhotoNim demo cam = <camType>
+    ./PhotoNim demo (perspective|orthogonal) [<output>] [--width=<width> --height=<height> --angle=<angle>]
 
 Options:
     --alpha=<alpha>     Color renormalization factor. [default: 0.18]
     --gamma=<gamma>     Gamma correction factor. [default: 1.0]
+    --width=<width>     Image wisth. [default: 1600]
```

Comment:
It could be fantastic if each command could display each options. For `pfm2png` one should pass alpha and gamma, while in the `demo` one should also pass width, height etc.